### DISCUSSION TITLE
Unreachable website with custom domain

### DISCUSSION THREAD

**Discussion by MariaSolOs** [vercel/community#5994] [Help]
### Summary

I have the `maria-sol-os.com domain`, which I purchased from Vercel. However, during the last few days the site has been down, despite me promptly paying the domain's renewal fees. Could you please help me troubleshoot this?

### Steps to Reproduce

Visit https://www.maria-sol-os.com/.

**Comment by swarnava** [comment]
Hi,

It looks like the domain is accessible already, can you confirm?

**Reply by MariaSolOs** [reply]
Still not working from my side:
![image](https://github.com/vercel/community/assets/62502207/e32c9ae4-2597-481e-b7c5-5cfa2e7343e6)

**Reply by swarnava** [reply]
I think you forgot to configure `www.maria-sol-os.com` and only configured `maria-sol-os.com` with `A` record. Can you double check under your domain? You will need to `www` subdomain with `CNAME` : https://dnschecker.org/#CNAME/www.maria-sol-os.com

**Reply by MariaSolOs** [reply]
Oh thanks! I just added the entry. I'll now wait for the changes to propagate...

![image](https://github.com/vercel/community/assets/62502207/317be09f-b341-4c06-999a-23f16d4e5724)

**Reply by swarnava** [reply]
You should set it to `cname.vercel-dns.com` : https://vercel.com/docs/projects/domains/troubleshooting#common-dns-issues

**Reply by MariaSolOs** [reply]
Oh that is already set, and has been for a very long time:
![image](https://github.com/vercel/community/assets/62502207/190414cd-2b0c-40e6-a022-f034e8818ca8)

**Reply by swarnava** [reply]
I mean 

Name `www`
Type `CNAME`
Value `cname.vercel-dns.com`

**Reply by MariaSolOs** [reply]
But isn't the star "matching" www?

**Reply by MariaSolOs** [reply]
> I mean
> 
> Name `www` Type `CNAME` Value `cname.vercel-dns.com`

@swarnava I've added this yet the site is still down from my end :( I've also tried clearing the browser's cache in case it was a local issue, but still nothing.

**Reply by swarnava** [reply]
Can you try on Incognito? I can see everything is setup correctly.

**Reply by MariaSolOs** [reply]
Yes, still nothing.

This is very strange because I've had this domain (purchased directly from Vercel) for years, and I haven't made any changes since then.

**Reply by swarnava** [reply]
Are you getting the same "Connection Refused" Error? Do you have VPN / Antivirus which maybe blocking this?

**Reply by MariaSolOs** [reply]
Yes, I have the same error. It also happens on my phone.

And no, I don't have anything like that.

**Comment by MariaSolOs** [comment]
Could I please get more assistance here? I've tried all suggestions from @swarnava but a week later I'm still experiencing this issue.

**Comment by MariaSolOs** [comment]
Because of this I will stop renewing my domain with Vercel. This lack of customer support is not worth it.

**Reply by swarnava** [reply]
Hi Maria, this discussion has totally got sidetracked with other community posts. Apologies. Could you post a screenshot of what you are seeing? I can access the website without any issue.

**Reply by MariaSolOs** [reply]
@swarnava I'm still having issues. I have tried clearing my browser cache, different browsers and computers, and I'm still getting an ERR_CONNECTION_REFUSED error when I visit the website.

This is the output I obtain when running `dns` on the terminal:
```
➜ dig ns maria-sol-os.com

; <<>> DiG 9.10.6 <<>> ns maria-sol-os.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 31290
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
; OPT=65401: 24 ("$")
; OPT=65402: 34 ("4")
;; QUESTION SECTION:
;maria-sol-os.com.              IN      NS

;; Query time: 26 msec
;; SERVER: 2607:f060:2::1#53(2607:f060:2::1)
;; WHEN: Sat Jun 15 14:12:14 PDT 2024
;; MSG SIZE  rcvd: 55
```

As you can see, the Vercel namespaces aren't on the response.

I've tried everything and I'm confused because I have another domain with the same configuration that is working fine.

**Comment by amyegan** [comment]
Hey @MariaSolOs. Like Swarnava, I'm able visit your site without any problem and the [NS check](https://www.whatsmydns.net/#NS/maria-sol-os.com) looks fine. But I do have a couple of ideas to investigate.

- I'm curious know if the problem is limited to your custom domain. What happens if you visit the [unique deployment URL](https://vercel.com/docs/deployments/generated-urls)?
- Does it work if you move the domain from your personal account to the team where your project is deployed?
  [Account Settings](https://vercel.com/account) > [Domains](https://vercel.com/account/domains) > click the three dot menu button next to your custom domain > choose "Move to a team" > follow the prompts in the popup to [move your domain](https://vercel.com/docs/projects/domains/transfer-your-domain)

**Comment by MariaSolOs** [comment]
Hello @amyegan. Thank you for helping.

- Yes, the issue is exclusive to my custom domain. https://me-mariasolos.vercel.app/ works just fine for me.
- I just tried moving the domain from my personal account to the team created for my projects. I'll wait a couple of days to see it that works.

**Reply by swarnava** [reply]
`ERR_CONNECTION_REFUSED` is usually ISP or client side block which related to company proxy / VPN/ Antivirus etc. Do you use any of those?


Also can you access the site if you switch DNS?

- Google DNS: https://developers.google.com/speed/public-dns
- Cloudflare DNS: https://www.cloudflare.com/en-gb/learning/dns/what-is-1.1.1.1/

**Reply by MariaSolOs** [reply]
I don't use any of those programs.

[a comment by an automated account is not shown]
